### PR TITLE
ASoC: A simple-card overlay for ADAU7002

### PR DESCRIPTION
**Pull request by plasticchris**
Usage: `dtoverlay=adau7002-simple`

### CODE REVIEW

**Comment by pelwell** [comment]
The overlay looks good - it's nice to see simple-card being useful for something. If you add an entry to the README I'll be happy to merge it.

Please observe the draconian README formatting rules - strict indentation, no tabs, two blank lines between entries, alphabetical order. You can either squash and force-push it or just push it as a separate commit for me to squash.

**Comment by pelwell** [comment]
And if you don't get any reaction after an update, add a comment - GitHub doesn't consistently notify us of updates.

**Comment by plasticchris** [comment]
Interestingly the pi 3 I was using runs out of memory attempting to force push: 

```
error: cannot fork() for pack-objects: Cannot allocate memory
fatal: git pack-objects failed: Cannot allocate memory
```

File: arch/arm/boot/dts/overlays/README [hunk 1]
```diff
@@ -168,6 +168,12 @@ Load:   dtoverlay=adau1977-adc
 Params: <None>
 
 
+Name:   adau7002-simple
+Info:   Overlay for the activation of ADAU7002 stereo PDM to I2S converter.
+Load:   dtoverlay=adau7002-simple
+Params: <None>
```
Comment (pelwell) [inline arch/arm/boot/dts/overlays/README]: But you do have a parameter - card-name.

File: arch/arm/boot/dts/overlays/README [hunk 2]
```diff
@@ -1360,7 +1366,6 @@ Params: led_gpio                GPIO for LED (default "17")
         led_trigger             Choose which activity the LED tracks (default
                                 "default-on")
 
-
```
Comment (pelwell) [inline arch/arm/boot/dts/overlays/README:1369]: Don't delete this line.

**Comment by pelwell** [comment]
I squashed and merged on the command line. Since I saw that the codec isn't supported by current kernel builds, I also added SND_SOC_ADAU7002 to the defconfigs.

**Comment by plasticchris** [comment]
Thanks; I had added it to my local config but missed the defconfigs.

**Pull request closed without merging**